CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.0.3 (Unreleased)
```

Comment:
This should just be `## Unreleased` as I would probably bump to `1.1.0` semantically as this includes new functionality that's backwards compatible, hence a minor version bump.

Reply:
Changing to unreleased per suggestion